causalpy/pymc_experiments.py

```diff
@@ -54,12 +54,23 @@ class TimeSeriesExperiment(ExperimentalDesign):
     def __init__(
         self,
         data: pd.DataFrame,
-        treatment_time: int,
+        treatment_time: int | float | pd.Timestamp,
```

Comment:
I suggest using `Union[int,float,pd.Timestamp]` to keep up the support of python 3.8

Reply:
Already made this change in a commit just now :)